From 308112b17f3d093c11cc25408a421c86364de828 Mon Sep 17 00:00:00 2001
From: Tony Cook <tony@develop-help.com>
Date: Tue, 17 Jan 2017 15:36:31 +1100
Subject: [PATCH] (perl #129149) fix the test so skip has a SKIP: to work with
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Thanks to bulk88 for pointing this out.

Petr Písař: Ported to 5.24.4 from:

From 30be69c851a7fa7e29d85c9b6e070273df82f3e7 Mon Sep 17 00:00:00 2001
From: Tony Cook <tony@develop-help.com>
Date: Tue, 17 Jan 2017 15:36:31 +1100
Subject: [PATCH] (perl #129149) fix the test so skip has a SKIP: to work with

Signed-off-by: Petr Písař <ppisar@redhat.com>
---
 t/op/pack.t | 1 +
 1 file changed, 1 insertion(+)

diff --git a/t/op/pack.t b/t/op/pack.t
index cf5ae78..e399f7e 100644
--- a/t/op/pack.t
+++ b/t/op/pack.t
@@ -2067,6 +2067,7 @@ SKIP:
 		    "integer overflow calculating allocation (multiply)");
 }
 
+SKIP:
 {
     # [perl #129149] the code below would write one past the end of the output
     # buffer, only detected by ASAN, not by valgrind
-- 
2.14.3

